### PR TITLE
Dunfell 3.3.0 recipes

### PR DESCRIPTION
**Pull request by oleorhagen**

### CODE REVIEW

File: meta-mender-core/recipes-mender/mender-connect/mender-connect_2.0.2.bb [hunk 1]
```diff
@@ -16,7 +16,7 @@ SRCREV = "949abff8750c58b8858800b4a15159db208f0f96"
 # Enable this in Betas, and in branches that cannot carry this major version as
 # default.
 # Downprioritize this recipe in version selections.
-# DEFAULT_PREFERENCE = "-1"
+DEFAULT_PREFERENCE = "-1"
```
Comment (kacf) [inline meta-mender-core/recipes-mender/mender-connect/mender-connect_2.0.2.bb:19]: Good catch! Important to remember this! 👍

**Review by kacf — APPROVED** [review]
Code looks good, but same question as in #1719.

**Comment by oleorhagen** [comment]
@mender-test-bot start pipeline

**Comment by oleorhagen** [comment]
@mender-test-bot start pipeline

**Comment by oleorhagen** [comment]
Missing the fix for https://github.com/mendersoftware/meta-mender/pull/1721 otherwise looks good to me

**Pull request merged**